common/views/components/ViewerSidebarPrototype/ViewerSidebarPrototype.tsx

```diff
@@ -223,16 +224,23 @@ const ViewerSidebarPrototype: FunctionComponent<Props> = ({
             <ViewerStructuresPrototype mainViewerRef={mainViewerRef} />
           </AccordionItem>
         )}
+        {/* // TODO only if search service available */}
         <AccordionItem title={'Search within this item'}>
           <Space v={{ size: 'm', properties: ['margin-bottom'] }}>
-            <TextInput
-              id={'test'}
-              type={'text'}
-              name={'test'}
-              label={'enter search term'}
-              value={inputValue}
-              setValue={setInputValue}
-            />
+            {itemViewerPrototypeWithSearch ? (
+              <IIIFSearchWithin mainViewerRef={mainViewerRef} />
+            ) : (
+              <Space v={{ size: 'm', properties: ['margin-bottom'] }}>
+                <TextInput
+                  id={'test'}
```

Comment:
we should probably update this `id` to something more meaningful now while we remember